dummyserver/testcase.py

```diff
@@ -292,6 +294,26 @@ class IPv6HTTPDummyProxyTestCase(HTTPDummyProxyTestCase):
     proxy_host_alt = "127.0.0.1"
 
 
+class HypercornDummyServerTestCase:
+    host: str = "localhost"
+    port: str = "8080"
```

Comment:
With Tornado, we use the port 0 (None in Python), but that does not seem to be possible with Hypercorn.

Reply:
That's a shame, can we open an issue on Hypercorn for this feature and link back to this?

Reply:
It should be, https://github.com/pgjones/hypercorn/issues/19

Reply:
Oh, right, thanks for the tip! I've addressed in https://github.com/urllib3/urllib3/pull/3190/commits/931c749e74ee68f670b11f80c8232e39a036dd2c by sneakily abusying `task_status.started` to overwrite `config.bind`.